### PR TITLE
Add links to alternative

### PR DESCRIPTION
**Pull request by ad-m**
Hello,

I am would like suggest to add information about alex/django-filter to README.rst, so users who can't use django-generic-filters will use django-filter.

I will pull requests note to django-filter.

 Greetings,

### CODE REVIEW

**Comment by mik-laj** [comment]
+1

File: README.rst [hunk 1]
```diff
@@ -116,3 +116,9 @@ Ressources
 * Code repository: https://github.com/novapost/django-generic-filters
 * Bugtracker: https://github.com/novapost/django-generic-filters/issues
 * Continuous integration: https://travis-ci.org/novapost/django-generic-filters
+
+**********
+Alternatives
+**********
+If the project does not meet your requirements - change it by pull requests or use an alternate `django-filter
+<https://github.com/alex/django-filter>`_.
```
Comment (tmartinfr) [inline README.rst]: Official GH page seems to be https://github.com/carltongibson/django-filter/. Could you update it?

**Comment by Natim** [comment]
@tmartinfr I think after a year we can make the changes ourselves.

**Comment by jpic** [comment]
@Natim It would be great to explain what's the difference between both.

**Pull request closed without merging**